ath-container.sh

```diff
@@ -13,11 +13,11 @@ java_version="${java_version:-17}"
 # high chance of uid / group already existing in the container
 # known to happen on macOS
 if ((uid < 1000)); then
-	uid=1001
+	uid=1000
```

Comment:
Recently we are deleting the default `ubuntu` user (UID 1000) inside the container, so no need to tiptoe around it.